### DISCUSSION TITLE
How can I setup the VS Solution and start the server?

### DISCUSSION THREAD

**Discussion by Bullsito** [OwlGamingCommunity/V#10] [Support]
I just download the repo and I try to open and compile the OwlV.sln and throws me so many errors about missing dependencies.

How can I do to make the solution working correctly and launching the server?

**Comment by 1Example** [comment]
It has been already solved in the existing discussions. Please check them.

**Reply by Bullsito** [reply]
Still doesn't working, I set it up but I don't have the cooker.exe and stills throwing me so many errors about missing dependencies

**Reply by 1Example** [reply]
Post the errors here if you are talking about server console.

**Reply by Bullsito** [reply]
They can't find some metadate files in some .dll (I think because the dependencies), I don't have libCodewalker.dll, Elasticsearh.net.dll, Nest.dll, SharpDX.dll, MySQLData.dll and SharpDX.Mathematics.dll.

These are the .dll:

owl_servershared.dll
owl_mysql.dll
owl_jobs.dll
owl_factions.dll
owl_core.dll
owl_account_system.dll
owl_vehicle_store.dll
owl_vehicles.dll
owl_vehicles.client-dll
owl_stores.dll
owl_stores.clients.dll
owl_scripter.dll
owl_radios.dll
owl_properties.dll
owl_items.dll
owl_items-client.dll
owl_http.dll
dvehic.dll
owl_donations.dll
owl_core.client.dll
owl_chat.dll
owl_banking.dll
owl_anticheat.dll
owl_admin.dll
owl_activities.dll
owl_achievements.dll


When I launch the RAGE MP Server console, this is the only error that is displayed, despite that, I can "connect" to the server (I spawn at the Observatory and I can't do anything else)

**Reply by Bullsito** [reply]
I was fixing aaaall the bad dependencies and I still having errors at libCodeWalker (Error at Using CodeWalker.GameFiles)

**Comment by Bullsito** [comment]
Okay, so here is the answer:

I just had to change every .csproj to netcoreapp3,1 and update all the dependencies through NuGet or manually from local files.

About libCodewalker, you had to recompile Vendor\LibCodewalker\libCodewalker.csproj

Now, I don't know If I did something else.

**Comment by Bullsito** [comment]
-- SOLVED --

I had these errors now:


Edit: I think It was the Sentry thing but I'm still having the issue but the log is change to this when I start the RAGE MP Server:

Could not load type 'System.Diagnostics.ExceptionExtentions' from assembly 'Ben.Demystifier, Version=0.4.0.0, Culture=neutral, PublicKeyToken=a6d206e05440431a'.
Unhandled exception. System.TypeLoadException: Could not load type 'System.Diagnostics.ExceptionExtentions' from assembly 'Ben.Demystifier, Version=0.4.0.0, Culture=neutral, PublicKeyToken=a6d206e05440431a'.
   at Bootstrapper.EventInternalClass.InitializeAPI(UInt32 sectionSize, UInt64* names, IntPtr method)
   
I'm also still having this error in my VS Terminal:

Unhandled exception 'System.TypeLoadException' at Unknown Module
Could not load type 'System.Diagnostics.ExceptionExtentions' from assembly 'Ben.Demystifier, Version=0.4.0.0, Culture=neutral, PublicKeyToken=a6d206e05440431a'.

----


Now that I solved the Ben.Demystifier error (I only had to downgrade everything to netcoreapp3.1 and the Ben.Demystifier.dll and System.Reflection.Metadata.dll from RAGEMP/Server-files/dotnet/runtime and I could launch the server I had the Connection Database error but I'll open a new discussion to try to fix it and don't mix themes.
